### ISSUE TITLE
Inspecting map within REBL goes too deep

### ISSUE THREAD

**Issue by luskwater** [cognitect-labs/REBL-distro#8]
[Symptom]
1. Start a REBL from the CLI: `clj -Rrebl -m cognitect.rebl`
2. Enter the REBL namespace: `(in-ns 'cognitect.rebl)`
3. In the REBL evaluation pane, enter a map inspection: `(cognitect.rebl/inspect {:a 2 :b "abc"})`

Results: the map is highlighted in the browse pane, but the view pane on the right shows the value of the first key: `1`
<img width="807" alt="screen shot 2018-12-04 at 1 02 31 pm" src="https://user-images.githubusercontent.com/42529/49463377-3df74280-f7c6-11e8-95a9-11a6281e3109.png">

**Update:** corrected names of panes in description

**Comment by stuarthalloway** [comment]
Hi Ron,

If you are running cognitect.rebl as your main function, you never need to call inspect.  You can just send the map directly. Is there a reason you are calling inspect?

Cheers,
Stu

**Comment by luskwater** [comment]
Yes, I'm requiring REBL in a CIDER session, and passing interesting things
to it as needed.
